testsuite/unittests/test_spike_det_reset.sli

```diff
@@ -22,17 +22,17 @@
 
 
  /** @BeginDocumentation
-Name: testsuite::test_spike_det_reset - test if resetting works on spike_detector
+Name: testsuite::test_spike_det_reset - test if resetting works on spike_recorder
```

Comment:
This file should probably be named `test_spike_rec_reset.sli`

Reply:
Done.